networking/src/Models/IGameSession.ts

```diff
@@ -14,6 +14,7 @@ export interface IGameSession {
     gameCode: number
     currentTimer?: number | null
     questions: Array<IQuestion>
+    title?: string | null
```

Comment:
What's `title`? Is this a new field needed for game session?

Reply:
Copied from game object I believe? I see the title field when I console log the response in the getGameSession API call - but when I call it in the use effect in the GameSessionContainer component I don't get access to the title field. I can see in AWS console that there is a game title field - so I know the object is already set up with it. But yea - couldn't figure out how to access it without adding the field in the IGameSession